_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-GSoC and
+ design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community to
+discuss the new major features that the contributor intends to implement during
+GSoC.
+
+**cEPs** are designed before implementation. Implementing major features
+usually happens from a technical perspective and may lead to poorly designed
+content - developer and user wise. This can essentially save a lot of coding
+work and also makes sure that major features are properly documented.
+
+Experience tells us that the best GSoC students are the ones who heavily
+contributed to the project ideas website, created projects from the issues,
+and expanded on existing ***.md*** files.
+
+So, if you are planning to do GSoC with coala, you should be starting to write
+detailed project ideas in our [projects](https://github.com/coala/projects)
+repository. Also, you must do enough research about the project in which you
+are interested to be able to express your ideas, timeline, etc clearly in your
+GSoC proposal which will increase your chances of success.
+
+For more information on cEP and to have an idea how to write a cEP,
```

Comment:
Something should come after `idea`, no?
Looks a little off. Maybe "about"? Or something?

Reply:
both "on" or "about" will look good !
what you suggest ? which will look more prominent ?

Reply:
I think, `cEP` is being repeated twice in the line.
Can we do something like: 
`For more information and to have an idea about how to write a cEP, ...`?

Reply:
Yep, we can avoid that repeating cEP word from the sentence.